kolibri/utils/cli.py

```diff
@@ -2,29 +2,37 @@
 
 import importlib  # noqa
 import logging  # noqa
+import os  # noqa
+import signal  # noqa
+import sys  # noqa
+
 # Do this before importing anything else, we need to add bundled requirements
 # from the distributed version in case it exists before importing anything
 # else.
-# TODO: Do we want to manage the path at an even more fundametal place like
+# TODO: Do we want to manage the path at an even more fundamental place like
 # kolibri.__init__ !? Load order will still matter...
-import os  # noqa
-import signal  # noqa
-import sys  # noqa
-from logging import config as logging_config  # noqa
 
 import kolibri  # noqa
 from kolibri import dist as kolibri_dist  # noqa
+sys.path = [
+    os.path.realpath(os.path.dirname(kolibri_dist.__file__))
+] + sys.path
 
-# Setup path in case we are running with dependencies bundled into Kolibri
-# (NOTE: This *must* come before imports below, of django etc, or whl/pex will fail)
-sys.path = [os.path.realpath(os.path.dirname(kolibri_dist.__file__))
-            ] + sys.path
+# Set default env
+os.environ.setdefault(
+    "DJANGO_SETTINGS_MODULE", "kolibri.deployment.default.settings.base"
+)
+os.environ.setdefault(
+    "KOLIBRI_HOME", os.path.join(os.path.expanduser("~"), ".kolibri")
+)
+os.environ.setdefault("KOLIBRI_LISTEN_PORT", "8008")
 
 import django  # noqa
 from django.core.management import call_command  # noqa
 from docopt import docopt  # noqa
 
 from . import server  # noqa
+from .system import become_daemon  # noqa
```

Comment:
Out of interest, did they just remove the functionality and not replace it with anything?